### ISSUE TITLE
Uncharted Vitagrafix error "your game version is not supported"

### ISSUE THREAD

**Issue by smoke20** [Electry/VitaGrafixPatchlist#103]
I have downloaded the latest patch files for Uncharted Golden Abyss and placed them under their respective location but when I start the game I get a Vitagrafix error " your game version is not supported". Is there something especific I need to do to get it to work or somehting I am mising? Thanks in advance.

**Comment by adrianbass360** [comment]
Yes, for this game you need an extra patch. If you look at the patchlist here on GitHub you'll see that at the bottom of the list there's a link to the patch required for this game to work with Vitagrafix.

**Comment by of-ravens-claw** [comment]
Golden Abyss's only supported version is `PCSF00001`, all of the other versions give that error.

Though it should be possible to rename the `PCSF00001.txt` file to another region's title id and it should just work.

EDIT: I just tried renaming the file with the US version `PCSA00029` and it worked just fine.
